swarm/api/routes/runs.py

```
@@ -1,2038 +1,139 @@
+    # Autopilot models
+    "AutopilotStartRequest",
+    "AutopilotStartResponse",
+    "AutopilotStatusResponse",
+    "AutopilotStopRequest",
+    "WisdomApplyResultResponse",
+    "_get_autopilot_controller",
+    # Issue models
+    "IssueIngestionRequest",
+    "IssueIngestionResponse",
+    "IssueSnapshot",
+]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)